net/openvswitch/Makefile

```diff
@@ -102,7 +102,7 @@ endef
 define Package/openvswitch-python
   $(call Package/openvswitch/Default)
   TITLE:=Open vSwitch Python Support
-  DEPENDS:=+openvswitch +PACKAGE_openvswitch-python:python +PACKAGE_openvswitch-python:python-six
+  DEPENDS:=+python +python-six
```

Comment:
so, according to the discussion via email with Felix, drop this commit ?

Reply:
Yes, I just reworked this to only remove openvswitch as a dependency while retaining the conditional deps

Reply:
ack